webvh/webvh/did/operations_manager.py

```diff
@@ -0,0 +1,496 @@
+            )
+
+    async def create(self, options: dict):
+        """Register identities."""
+
+        server_url = get_server_info(self.profile)
+
+        namespace = options.get("namespace")
+
+        if namespace is None:
+            raise DidCreationError("Namespace is required.")
+
+        # Generate a random identifier if not provided
+        identifier = options.get("identifier", str(uuid4()))
+
+        # Contact the server to request an identifier
+        did, challenge, domain, expiration = await self._request_identifier(
+            server_url, namespace, identifier
+        )
+
+        author_key_info = await self._get_or_create_author_key(did)
+        author_secured_document = await self._create_author_signed_registration_document(
+            did, author_key_info, expiration, domain, challenge
+        )
+
+        parameters = options.get("parameters", {})
+        result = await EndorsementManager(self.profile).endorse_registration_document(
+            author_secured_document, expiration, domain, challenge, parameters
+        )
+
+        if isinstance(result, dict):
+            return await self.finish_create(
+                result,
+                parameters=parameters,
+                state=RegistrationState.SUCCESS.value,
+                author_key_info=author_key_info,
+            )
+
+        try:
+            return await asyncio.wait_for(
+                self._wait_for_endorsement(did),
+                ENDORSEMENT_WAIT_TIMEOUT_SECONDS,
+            )
+        except asyncio.TimeoutError:
+            return {
+                "status": "unknown",
+                "message": "No immediate response from endorser agent.",
+            }
+
+    async def _create_signed_initial_log_entry(
+        self,
+        session: ProfileSession,
+        domain: str,
+        namespace: str,
+        identifier: str,
+        parameters: dict,
+        author_key_info: dict,
+    ):
+        initial_doc = {
+            "@context": "https://www.w3.org/ns/did/v1",
+            "id": r"did:webvh:{SCID}:" + domain + ":" + namespace + ":" + identifier,
+        }
+        doc_state = DocumentState.initial(
+            {
+                "updateKeys": [author_key_info.get("multikey")],
+                "prerotation": parameters.get("prerotation", False),
+                "portable": parameters.get("portable", False),
+                "method": WEBVH_METHOD,
+            },
+            initial_doc,
+        )
+
+        # Add author proof to the log entry
+        return await DataIntegrityManager(session).add_proof(
+            doc_state.history_line(),
+            DataIntegrityProofOptions(
+                type="DataIntegrityProof",
+                cryptosuite="eddsa-jcs-2022",
+                proof_purpose="assertionMethod",
+                verification_method=f"did:key:{author_key_info.get('multikey')}#{author_key_info.get('multikey')}",
```

Comment:
I would rename `author` with `authorized` or `update` to use the webvh spec linguo